app/com/lynxanalytics/biggraph/graph_operations/ExecuteSQL.scala

```diff
@@ -32,7 +31,10 @@ object ExecuteSQL extends OpFromJson {
     val functionRegistry = FunctionRegistry.builtin
     val reg = UDFHelper.udfRegistration(functionRegistry)
     UDF.register(reg)
-    val catalog = new SessionCatalog(new InMemoryCatalog, functionRegistry, sqlConf)
+    val catalog = new spark.sql.catalyst.catalog.SessionCatalog(
```

Comment:
This class is abstract on Databricks. The seemingly meaningless change here makes it easier to replace the class name when building for Databricks.